airflow/jobs/backfill_job.py

```diff
@@ -471,6 +476,14 @@ def _per_task_process(key, ti, session=None):
                             ti_status.running.pop(key)
                         # Reset the failed task in backfill to scheduled state
                         ti.set_state(State.SCHEDULED, session=session)
+                elif self.rerun_succeeded_tasks and ti.state == State.SUCCESS:
+                    # Rerun succeeded tasks
+                    self.log.error("Task instance {ti} with state {state}".format(ti=ti,
+                                                                   state=ti.state))
```

Comment:
```suggestion
                    self.log.info("Task instance %s with state %s succeeded", ti,ti.state)
```
This should not be an error log, right?